devops/azure-pipelines/gradle-build-test-stage-template.yml

```diff
@@ -17,4 +17,7 @@ stages:
         spotBugsAnalysis: false
   - template: ./discord-job-template.yml
     parameters:
-      dependOn: Build_and_Test  
\ No newline at end of file
+      dependOn: Build_and_Test
+      name: 'Team2-P3'
+      title: 'Team 2 Project 3 Backend'
+      discordVarGroup: Discord-Webhook-Info
```

Comment:
this file, if it's a template, should also be generic and not itself include these values, unless you have some good justification.
you may or may not consider changing your overall template structure